library/components/TPager.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div>
+    <div v-if="numberOfItems > itemsPerPage" class="center">
+      <span
+        ref="AngleDoubleLeft"
```

Comment:
Though not really a big deal, but I'd probably use this convention `angleDoubleLeft `.

Reply:
WTF was I thinking, good catch! 😂 Fixed.